### DISCUSSION TITLE
umiMax 如何配置清除控制台日志打印

### DISCUSSION THREAD

**Discussion by shaw-Lee** [umijs/umi#10643] [Q&A] [answered]
umiMax 如何配置清除控制台日志打印，就像umi3.x配置一样
// 生产环境去掉控制台日志
    terserOptions: {
        compress: {
            drop_console: isEnvProduction
        }
    },

**Comment by fz6m** [comment, marked as answer]
```ts
  jsMinifierOptions: {
    drop: ['console', 'debugger']
  }
```

 - [jsMinifierOptions](https://umijs.org/docs/api/config#jsminifieroptions)
